core/roboconf-dm/src/main/java/net/roboconf/dm/management/Manager.java

```diff
@@ -379,13 +342,16 @@ public void reconfigure() {
 		}
 
 		// Restart the templating, using the new configuration directory.
-		synchronized (this) {
-			if( this.templatingManager != null ) {
-				this.templatingManager.stopTemplating();
-				this.templatingManager.startTemplating( this.configurationDirectory );
-				// Applications are re-added as they are reloaded, in the restoreApplications() method.
-			}
-		}
+		// FIXME: will be removed once we implement #394
```

Comment:
??? Is #394 fixed yet?
AFAIK it isn't, so why are those lines ~~removed~~ commented out?